### PR TITLE
Add initial KG building functions.

### PR DESCRIPTION
**Pull request by jpalowitch**

### CODE REVIEW

**Review by tzufgoogle — CHANGES_REQUESTED** [review]
Just a few things: 
(1) typing wrong (leads to a build error)
(2) a function without typing
(3) maybe add a test for get_geofenced_wikidata_relations?

File: cabby/data/wikidata/query.py [hunk 1]
```diff
@@ -158,7 +249,84 @@ def get_geofenced_wikidata_items(region: Text) -> Sequence[Dict[Text, Any]]:
       filtered_results.append(result)
   return filtered_results
 
+def get_filter_string(place_filter: [Sequence[Text]],
```
[the remark this answers is not in the document]
Comment (jpalowitch) [reply cabby/data/wikidata/query.py:252]: Fixed by 54ba279

File: cabby/data/wikidata/query.py [hunk 2]
```diff
@@ -158,7 +249,84 @@ def get_geofenced_wikidata_items(region: Text) -> Sequence[Dict[Text, Any]]:
...
+  """
+  if len(place_filter) == 0:
+    return ""
+  filter_string = "FILTER (?%s IN (%s))" % (
+    place_param,
+    ",".join(["wd:%s" % qid for qid in place_filter]))
+  return filter_string
+
+def get_locations_by_qid(region: Text,
+                         place_filter: [Sequence[Text]] = []):
```
[the remark this answers is not in the document]
Comment (jpalowitch) [reply cabby/data/wikidata/query.py]: Fixed by 54ba279

File: cabby/data/wikidata/query.py [hunk 3]
```diff
@@ -158,7 +249,84 @@ def get_geofenced_wikidata_items(region: Text) -> Sequence[Dict[Text, Any]]:
...
+  query_result = query_api(query)
+  locations = {}
+  for result in query_result:
+    qid = result['place']['value'].rsplit("/", 1)[1]
+    point = util.point_str_to_shapely_point(result['location']['value'])
+    locations[qid] = point
+  return locations
+
+def get_geofenced_wikidata_relations(region: Text,
+                                     place_filter: [Sequence[Text]] = [],
```
[the remark this answers is not in the document]
Comment (jpalowitch) [reply cabby/data/wikidata/query.py]: Fixed by 54ba279

File: cabby/data/metagraph/utils.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+# limitations under the License.
+
+'''Utils for metagraph construction'''
+
+import networkx as nx
+import pandas as pd
+
+def convert_pandas_df_to_metagraph(df, source_column, target_column,
+                                   source_metadata_columns, target_metadata_columns,
+                                   edge_attribute_columns):
```
[the remark this answers is not in the document]
Comment (jpalowitch) [reply cabby/data/metagraph/utils.py]: Fixed by 54ba279

File: cabby/data/metagraph/utils.py [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Utils for metagraph construction'''
+
+import networkx as nx
+import pandas as pd
+
+def convert_pandas_df_to_metagraph(df, source_column, target_column,
```
[the remark this answers is not in the document]
Comment (jpalowitch) [reply cabby/data/metagraph/utils.py]: Fixed by 54ba279

**Review by tzufgoogle — APPROVED** [review]
Let's merge this

File: cabby/data/wikidata/query.py [hunk 6]
```diff
@@ -196,17 +237,94 @@ def get_geofenced_wikidata_items(region: Text) -> Sequence[Dict[Text, Any]]:
       filtered_results.append(result)
   return filtered_results
 
+def get_filter_string(place_filter: [Sequence[Text]],
```
Comment (tzufgoogle) [inline cabby/data/wikidata/query.py:240]: Change: [Sequence[Text]] => Sequence[Text]

File: cabby/data/wikidata/query_test.py [hunk 7]
```diff
@@ -37,6 +37,11 @@ def testSingleOutputWithoutInstance(self):
     poi_by_value = [x['placeLabel']['value'] for x in output]
     self.assertIn(expected_place_label, poi_by_value)
 
+  def testRelationsQuery(self):
+    wd_relations = query.get_geofenced_wikidata_relations(
+      "Pittsburgh", extract_qids=True)
+    qid_set = set(list(wd_relations.place))
```
Comment (tzufgoogle) [inline cabby/data/wikidata/query_test.py:43]: wd_relations doesn't have place

**Review by tzufgoogle — CHANGES_REQUESTED** [review]
The re are some building errors and test errors. 
Run:
(1) source build_all.sh
(2) source test_all.sh

File: cabby/data/wikidata/query.py (hunk 6 above)
Comment (jpalowitch) [reply cabby/data/wikidata/query.py]: Fixed by 49c50b2

File: cabby/data/wikidata/query_test.py (hunk 7 above)
Comment (jpalowitch) [reply cabby/data/wikidata/query_test.py:43]: Fixed by 49d6a81

**Review by tzufgoogle — APPROVED** [review]
LGTM

**Pull request merged**